src/ctrl.c

```diff
@@ -59,6 +60,8 @@ gboolean terminal_accept_cb(int fd, G_GNUC_UNUSED GIOCondition condition, G_GNUC
 	masterfd_stdin = console.fd;
 	masterfd_stdout = console.fd;
 
+	schedule_master_stdin_write();
```

Comment:
could you add a comment motivating this change? just so future developers know why we need to wait till here to add the fd

Reply:
sure, added a comment